contents/tutorials/python-feature-flags.md

```diff
@@ -0,0 +1,169 @@
+---
+title: How to set up Python feature flags in Flask
+date: 2023-09-04
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-18.png
+tags: ["feature-flags"]
+---
+
+Feature flags make it easy to conditionally run code and show content based on users or conditions. In this tutorial, we show how to create a basic Python Flask app, add PostHog, and set up feature flags to conditionally show content in the app.
+
+## Creating a Flask app and adding PostHog
+
+To showcase [feature flags](/docs/feature-flags), we use Flask, a Python web framework, to create an app. To start, we create a folder for our app named `flag-demo` and create a file named `hello.py` in that folder.
+
+```bash
+mkdir flag-demo
+cd flag-demo
+touch hello.py
+```
+
+Next, create a virtual environment named `venv` for our app, activate the virtual environment, and install Flask.
+
+```bash
+python3 -m venv venv
+. venv/bin/activate
+pip install Flask
+```
+
+In `hello.py`, create a home route that returns a basic "Hello, World!" and a `/user/<string:user>` route that returns the user string like this:
+
+```python
+# flag-demo/hello.py
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def hello_world():
+  return "<p>Hello, World!</p>"
+
+@app.route("/user/<string:user>")
+def show_user(user):
+    return f"<p>Hello, {user}!</p>"
+```
+
+Finally run `flask --app hello run` and go to [`http://127.0.0.1:5000`](http://127.0.0.1:5000/) to see your basic app running.
```

Comment:
This didn't render correclty. I think you have to have the hyperlinks inside the code.
```suggestion
Finally run `flask --app hello run` and go to `[http://127.0.0.1:5000](http://127.0.0.1:5000/)` to see your basic app running.
```